getting_started/setup_vm/roles/openenclave/vars/common.yml

```diff
@@ -1,6 +1,6 @@
-oe_ver: "0.18.2"
+oe_ver: "0.18.4"
```

Comment:
Update changelog too?

Reply:
Will do in the PR that actually changes the cmake, deb and yaml files.